runbot_travis2docker/models/runbot_branch.py

```diff
@@ -5,12 +5,47 @@
 
 import re
 
-import requests
 import subprocess
+import requests
 
 from openerp import fields, models, api
 
 
+def read_group(branch):
```

Comment:
Better use `ormcache`
Use the following PR to get a example of the `How to`:
https://github.com/Vauxoo/enterprise/pull/35